psscriptanalyzer/Test-Connection.ps1

```
@@ -0,0 +1,52 @@
+param(
+    [string]$ComputerName = "localhost",
+    [string]$Username = "demoapp-svc",
+    [string]$Password = "Sv1cP@ss2024"
```

PSScriptAnalyzer (1.25.0)

[warning] Parameter '$Password' should not use String type but either SecureString or PSCredential, otherwise it increases the chance to expose this sensitive information.

Suggested fix: Set $Password type to SecureString; Set $Password type to PSCredential

(PSAvoidUsingPlainTextForPassword)